extension/spark-doris-connector/src/main/java/org/apache/doris/spark/serialization/RowBatch.java

```diff
@@ -70,7 +70,8 @@ public void put(Object o) {
         }
     }
 
-    private int offsetInOneBatch = 0;
+    // offset for iterate the rowBatch
+    private int offsetInRowBatch = 0;
     private int rowCountInOneBatch = 0;
     private int readRowCount = 0;
     private List<Row> rowBatch = new ArrayList<>();
```

Comment:
whether we can use a fixed array which size = rowCount?

Reply:
We could get the fixed array size by add all rows of Arrow batch through `arrowStreamReader.loadNextBatch()`, in this case, we might as well use List as we do now